nsv13/code/modules/overmap/fighters/fighters.dm

```diff
@@ -994,6 +1031,257 @@ After going through this checklist, you're ready to go!
+Throttle lock on
+Disengage battery
+Disengage fuel pump (or engine gets flooded)
+Turn off ignition
+
+If you run out of fuel:
+Activate the brakes and begin a shutdown of your fighter. Once you have received more fuel, begin startup sequence as expected. If you run out of fuel, you will be stuck adrift. It is highly recommended that you RTB when you hit 100 fuel as you'll have 30 seconds or so more burn time before you fizzle out.
+
+How to make fuel:
+1 part hydrogen : 1 part carbon to make hydrocarbon. Mix hydrocarbon and welding fuel to produce tyrosene
+
+*/
+
+/obj/structure/overmap/fighter/proc/toggle_canopy()
+	canopy_open = !canopy_open
+	playsound(src, 'nsv13/sound/effects/fighters/canopy.ogg', 100, 1)
+
+/obj/structure/overmap/fighter/verb/show_control_panel()
+	set name = "Show control panel"
+	set category = "Ship"
+	set src = usr.loc
+
+	if(!verb_check())
+		return
+	ui_interact(usr)
+
+/obj/structure/overmap/fighter/ui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = 0, datum/tgui/master_ui = null, datum/ui_state/state = GLOB.default_state) // Remember to use the appropriate state.
+	if(user != pilot)
+		return
+	ui = SStgui.try_update_ui(user, src, ui_key, ui, force_open)
+	if(!ui)
+		ui = new(user, src, ui_key, "fighter_controls", name, 560, 600, master_ui, state)
+		ui.open()
+
+/obj/structure/overmap/fighter/can_move()
+	var/obj/item/twohanded/required/fighter_component/engine/engine = get_part(/obj/item/twohanded/required/fighter_component/engine)
+	if(!engine)
+		if(pilot)
+			to_chat(pilot, "<span class='warning'>WARNING: This fighter doesn't have any engines!</span>")
+		return FALSE
+	if(flight_state == NO_FUEL)
+		return FALSE
+	if(mag_lock)
+		return FALSE
+	if(flight_state != FLIGHT_READY || throttle_lock)
+		return FALSE
+	return TRUE
+
+/obj/structure/overmap/fighter/proc/check_start() //See if we can kick off the engine off of the APU.
+	if(user_thrust_dir)
+		playsound(src, 'nsv13/sound/effects/fighters/startup.ogg', 100, FALSE)
+		visible_message("<span class='warning'>[src]'s engine bursts into life!</span>")
+		flight_state = FLIGHT_READY
+		add_overlay("engine_start")
+		relay('nsv13/sound/effects/fighters/cockpit.ogg', "<span class='warning'>You hear a loud noise as [src]'s engine kicks in.</span>", loop=TRUE, channel = CHANNEL_SHIP_ALERT)
+		return TRUE
+	relay('nsv13/sound/effects/fighters/master_caution.ogg', "<span class='warning'>WARNING: Engine ignition failure.</span>")
+	playsound(src, 'nsv13/sound/effects/ship/rcs.ogg', 100, TRUE)
+	visible_message("<span class='warning'>[src]'s engine fizzles out!</span>")
+	flight_state = NO_IGNITION
+	return FALSE
+
+/obj/structure/overmap/fighter/ui_act(action, params, datum/tgui/ui)
+	if(..())
+		return
+	if(!in_range(src, usr) || !pilot || usr != pilot) //Topic check
+		return
+	if(warmup_cooldown)
+		to_chat(usr, "You need to wait for [src] to finish its last action.</span>")
+		return
+	switch(action)
+		if("ignition")

```

Comment:
I feel like this all might end up being too much interaction required for the user. It may end up being better tp just have manual turn on setup by the flight controller long term. The launch itself powers up for flight only.